.github/actions/releases/setup-android/action.yml

```
@@ -32,6 +32,13 @@ runs:
 
     - name: Setup Android SDK
       uses: android-actions/setup-android@v3
+      
+    # "Fixes firebase build error when no valid google-services.json is present"
+    - name: Comment out Google Play Services import
+      shell: bash
+      run: |
+        sed -i 's/id .com\.google\.gms\.google-services./\/\/ id .com\.google\.gms\.google-services./' android/app/build.gradle
+        echo "Google Play Services import commented out"
```

Release builds should fail if Firebase is not configured correctly. I will fix the root cause shortly. This also happens with macOS or iOS (can't recall which it was).

Please also create tasks for (normal priority):

Ensure this behaviour is consistent across platforms.
Amend our Firebase local setup documentation to include instructions and/or a script in our CLI package script to "de-Google" the builds.

---

macOS builds succeed without Firebase configs, but the app crashes on startup with a Firebase-related runtime exception. 

Android builds are the only builds failing because of firebase configs currently, and because of the introduction of the `com.google.gms.google-services` import in this PR. The new import was added by `flutterfire_cli`.

All platforms, except for macOS, worked fine without valid firebase configs present - the runtime error appears to only be fatal on macOS. Linux is also not yet supported by `flutterfire_cli`/firebase.